addons/website_slides/tests/test_robots.py

```diff
@@ -0,0 +1,20 @@
+from odoo.addons.website.tests.test_robots_txt import WebsiteRobotsTxtTestCommon
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TestWebsiteSlidesRobots(WebsiteRobotsTxtTestCommon):
+    """Test robots.txt directives for website_slides addon"""
+
+    def test_slides_tag_pages_disallowed(self):
+        """Test that slides tag pages and filters are disallowed"""
+        self.assertRobotsTxtValues({
+            '*': {
+                'disallow': [
+                    '/slides/all/tag/',
+
```

Comment:
only test on for website or website_test or ... 
don't test on each module that the code of robots.txt work, if it work for one, it work for all...

When you test a thing on sale order, you don't test on each id of the db, only on a random so :fearful: